### ISSUE TITLE
environment.yaml or requirements.txt for preparing train environment

### ISSUE THREAD

**Issue by weivnaeco** [NVlabs/FocalFormer3D#8]
Hello. @chenyilun95 
Even I followed the steps in install.md, I keep getting errors when I try to run "python tools/train.py DeformFormer3D_L.py" on NVIDIA GeForce RTX 3090.
Could you please pack the conda env for me using "conda env export > environment.yaml" or 
"pip freeze > requirements.txt, pip list --format=freeze > requirements.txt"?
Thank you. Have a nice day.

**Comment by chenyilun95** [comment]
Sure, as following:

> absl-py==1.4.0
addict==2.4.0
anyio==3.7.1
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
arrow==1.2.3
asttokens==2.2.1
attrs==23.1.0
backcall==0.2.0
beautifulsoup4==4.12.2
black==23.3.0
bleach==6.0.0
cachetools==5.3.1
certifi==2023.5.7
cffi==1.15.1
charset-normalizer==3.1.0
click==8.1.4
comm==0.1.3
contourpy==1.1.0
cycler==0.11.0
debugpy==1.6.7
decorator==5.1.1
defusedxml==0.7.1
descartes==1.1.0
exceptiongroup==1.1.2
executing==1.2.0
fastjsonschema==2.17.1
filelock==3.12.2
fire==0.5.0
flake8==6.0.0
fonttools==4.40.0
fqdn==1.5.1
fsspec==2023.6.0
fvcore==0.1.5.post20210915
google-auth==2.21.0
google-auth-oauthlib==1.0.0
grpcio==1.56.0
huggingface-hub==0.16.4
idna==3.4
imageio==2.31.1
importlib-metadata==6.7.0
importlib-resources==5.12.0
iniconfig==2.0.0
iopath==0.1.9
ipykernel==6.24.0
ipython==8.12.2
ipython-genutils==0.2.0
ipywidgets==8.0.7
isoduration==20.11.0
jedi==0.18.2
Jinja2==3.1.2
joblib==1.3.1
jsonpointer==2.4
jsonschema==4.18.0
jsonschema-specifications==2023.6.1
jupyter==1.0.0
jupyter-console==6.6.3
jupyter-events==0.6.3
jupyter_client==8.3.0
jupyter_core==5.3.1
jupyter_server==2.7.0
jupyter_server_terminals==0.4.4
jupyterlab-pygments==0.2.2
jupyterlab-widgets==3.0.8
kiwisolver==1.4.4
llvmlite==0.31.0
lyft-dataset-sdk==0.0.8
Markdown==3.4.3
markdown-it-py==3.0.0
MarkupSafe==2.1.3
matplotlib==3.5.2
matplotlib-inline==0.1.6
mccabe==0.7.0
mdurl==0.1.2
mistune==3.0.1
mmcv==1.3.18
mmcv-full==1.4.0
mmdet==2.14.0
-e git+https://github.com/open-mmlab/mmdetection3d.git@f1107977dfd26155fc1f83779ee6535d2468f449#egg=mmdet3d
mmengine==0.7.4
mmsegmentation==0.14.1
model-index==0.1.11
mypy-extensions==1.0.0
nbclassic==1.0.0
nbclient==0.8.0
nbconvert==7.6.0
nbformat==5.9.0
nest-asyncio==1.5.6
networkx==2.2
notebook==6.5.4
notebook_shim==0.2.3
numba==0.48.0
numpy==1.19.5
nuscenes-devkit==1.1.10
oauthlib==3.2.2
opencv-python==4.8.0.74
opendatalab==0.0.9
openmim==0.3.9
ordered-set==4.1.0
overrides==7.3.1
packaging==23.1
pandas==1.4.4
pandocfilters==1.5.0
parso==0.8.3
pathspec==0.11.1
pexpect==4.8.0
pickleshare==0.7.5
Pillow==10.0.0
pkgutil_resolve_name==1.3.10
platformdirs==3.8.1
plotly==5.15.0
pluggy==1.2.0
plyfile==0.9
portalocker @ file:///croot/portalocker_1682958299664/work
prettytable==3.8.0
prometheus-client==0.17.0
prompt-toolkit==3.0.39
protobuf==4.23.4
psutil==5.9.5
ptyprocess==0.7.0
pure-eval==0.2.2
pyasn1==0.5.0
pyasn1-modules==0.3.0
pycocotools==2.0.6
pycodestyle==2.10.0
pycparser==2.21
pycryptodome==3.18.0
pyflakes==3.0.1
Pygments==2.15.1
pyparsing==3.0.9
pyquaternion==0.9.9
pytest==7.4.0
python-dateutil==2.8.2
python-json-logger==2.0.7
pytorch3d==0.7.2
pytz==2023.3
PyWavelets==1.4.1
PyYAML @ file:///croot/pyyaml_1670514731622/work
pyzmq==25.1.0
qtconsole==5.4.3
QtPy==2.3.1
referencing==0.29.1
requests==2.31.0
requests-oauthlib==1.3.1
rfc3339-validator==0.1.4
rfc3986-validator==0.1.1
rich==13.4.2
rpds-py==0.8.8
rsa==4.9
safetensors==0.3.1
scikit-image==0.19.3
scikit-learn==1.3.0
scipy==1.10.1
Send2Trash==1.8.2
Shapely==1.8.5
six==1.16.0
sniffio==1.3.0
soupsieve==2.4.1
stack-data==0.6.2
tabulate @ file:///opt/conda/conda-bld/tabulate_1657784105888/work
tenacity==8.2.2
tensorboard==2.13.0
tensorboard-data-server==0.7.1
termcolor @ file:///croot/termcolor_1668084651543/work
terminado==0.17.1
terminaltables==3.1.10
threadpoolctl==3.1.0
tifffile==2023.7.4
timm==0.9.2
tinycss2==1.2.1
tomli==2.0.1
torch==1.9.1+cu111
torchvision==0.10.1+cu111
tornado==6.3.2
tqdm @ file:///croot/tqdm_1679561862951/work
traitlets==5.9.0
trimesh==2.35.39
typing_extensions==4.7.1
tzdata==2023.3
uri-template==1.3.0
urllib3==1.26.16
wcwidth==0.2.6
webcolors==1.13
webencodings==0.5.1
websocket-client==1.6.1
Werkzeug==2.3.6
widgetsnbextension==4.0.8
yacs @ file:///tmp/build/80754af9/yacs_1634047592950/work
yapf==0.40.1
zipp==3.15.0
